backends/p4tools/modules/testgen/CMakeLists.txt

```diff
@@ -52,15 +52,7 @@ set(
   test/gtest_utils.cpp
   test/lib/format_int.cpp
   test/lib/taint.cpp
-  test/small-step/binary.cpp
```

Comment:
Do these all assume `v1model` in some way? Might be a flaw in the test design...

Reply:
They all fail with messages like this:

```
C++ exception with description "In file: /home/xstill/repo/p4c/backends/p4tools/modules/testgen/test/gtest_utils.cpp:21
Compiler Bug: Target bmv2/v1model not supported
" thrown in the test body.
```

Frankly, I'm not entirely sure what is the relation between BMV2 and v1model here :-).

Reply:
Hmmm, maybe we should introduce a `p4test` like extension that works for any target at some point. Just for debugging. 

The `v1model1` arch is executed on `bmv2`. Theoretically it could also be `v1model` on DPDK or Tofino.

Reply:
I would replace "theoretically" with "hypothetically", or any other word that means "this seems very unlikely to happen, until and unless some very time-rich and motivated volunteers agree to extend P4-DPDK open source implementation to do this".

It is open source, so of course it can happen.  I just don't want anyone reading this to mistake the prerequisites required to make it happen.  No one I am aware of is signing up for such a project for P4-DPDK.